maintainers/maintainer-list.nix

```diff
@@ -3947,6 +3947,13 @@
     githubId = 17111639;
     name = "Devin Singh";
   };
+  devpikachu = {
```

Comment:
Would you please do this in a separate commit titled `maintainers: add devpikachu`? Thanks!

Reply:
https://github.com/NixOS/nixpkgs/pull/239567